KMB/make_KMB_info.py

```diff
@@ -8,13 +8,17 @@
 """
 from __future__ import unicode_literals
 from collections import OrderedDict
+import os.path
 
 import pywikibot
+from pywikibot.data import sparql
+
 import batchupload.helpers as helpers
 import batchupload.common as common
 from batchupload.make_info import MakeBaseInfo
 
 
+MAPPINGS_DIR = 'mappings'
 BATCH_CAT = 'Media contributed by RAÄ'  # stem for maintenance categories
```

Comment:
In my view the two are not part of the same category tree. `Images from the Swedish National Heritage Board` is related to the image source. So when I upload an image that I found on their page it should go there.

`Media contributed by RAÄ` is related to the upload project, i.e. that we did this s a partnership with them. This is where the category for the batch (`Media contributed by RAÄ: 2017-05`) is found along with any maintenance categories. Each image would in turn also be categorised as  `Images from the Swedish National Heritage Board` (which happens through the template)